scripts/collections-manager

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/env python
+import glob
+import os
+import subprocess
+import sys
+import tempfile
+import time
+from contextlib import contextmanager
+from importlib import import_module
+from pathlib import Path
+from typing import Generator, List
+
+import typer
+import yaml
+
+DEFAULT_ROOT = Path("~/src/github.com/PrefectHQ").expanduser()
```

Comment:
If you use the `GOPATH` style, you'll have the least friction here and won't have to specify this on every command.  We could also have this come from an envvar too...